src/Data/Textile/WellKnown.elm

```diff
@@ -96,6 +97,7 @@ load processes =
         |> fromAlias "dyeing-fabric"
         |> fromAlias "dyeing-synthetic-fiber"
         |> fromAlias "dyeing-yarn"
+        |> fromAlias "elec-medium-region-asia"
```

Comment:
It looks like we are using aliases as primary keys here, are we ok with that? Shouldn't we use ids instead?

Reply:
That's exactly the purpose of aliases: identifying well known processes for internal use with explicit names.

Reply:
Well, sure, but if we rely on them in the code, we should ensure that they are unique and not empty from the data pipeline perspective (this is something that is not enforced at the moment).
And for me, using aliases as PKs seem to defeat the purpose of the whole move from slug to uuids for the `id` field, but I may be missing something here.

Reply:
Ok, please file a dedicated issue about that, this is definitely not the scope of this card

Reply:
Or is it ? 😩

Reply:
after discussing it over mm, it's been decided to switch to relying on uuids in the code but in a next dedicated PR.